gum/backend-arm64/guminterceptor-arm64-glue.S

```diff
@@ -23,58 +23,86 @@
   sub sp, sp, GUM_FRAME_SIZE - (2 * 8)
 
   # We want to clobber x1, so let's save it (and x2) first
-  stp x1, x2, [sp, GUM_PADDING_SIZE + (3 * 8)]
+  stp x1, x2, [sp, GUM_PADDING_SIZE + (4 * 8)]
+
+  # sample nzcv
+  mrs x1, nzcv
```

Comment:
i'm pretty sure `nzcv` is partially clobbered for the previous `sub`... not sure how to do better 🤔

Reply:
This is fine since `sub` doesn't clobber, but `subs` does. (If this later turns out to be incorrect we could always use `x16` as a scratch register, since it's already saved.)